contractcourt/channel_arbitrator.go

```diff
@@ -1131,11 +1139,102 @@ func (c *ChannelArbitrator) sweepAnchors(anchors []*lnwallet.AnchorResolution,
 		if err != nil {
 			return err
 		}
+
+		return nil
+	}
+
+	// Calculate the deadline. This is only valid for sweeping the local
+	// anchor.
+	deadline, err := c.findCommitmentDeadline(heightHint)
+	if err != nil {
+		return err
+	}
+
+	for _, anchor := range anchors {
+		// The first item is the anchor of the local commitment tx, we
```

Comment:
Logic will likely change w/ the move to the struct to make things more explicit.

Reply:
Changed.